backend/config/curation-api.yml

```diff
@@ -862,30 +989,36 @@ components:
           type: string
       type: object
     processing_status_collection:
-      description: >-
+      description: |
         Processing status for a Collection
-
-        * `"FAILURE"`: one or more Datasets ran into an error during processing. The CELLxGENE team will reach out to help you.
-        * `"PENDING"`: one or more Datasets are still being processed.
-        * `"SUCCESS"`: all Datasets have been processed successfully.
-        * `null`: the Collection has no Datasets.
+        | `processing_status` | Description |
+        | ------------------- | ----------- |
+        | `"FAILURE"` | one or more Datasets ran into an error during processing. More information can be found in the Dataset processing_status logs. |
+        | `"PENDING"` | one or more Datasets are still being processed. |
+        | `"SUCCESS"` | all Datasets have been processed successfully. |
+        | `null` | the Collection has no Datasets. |
       enum:
         - FAILURE
         - PENDING
         - SUCCESS
       nullable: true
       type: string
     processing_status_dataset:
-      description: >-
+      description: |
         Processing status for a Dataset
-
-          * `"FAILURE"`: the Dataset ran into an error during processing. The CELLxGENE team will reach out to help you.
-          * `"PENDING"`: the Dataset has is still being processed.
-          * `"SUCCESS"`: the Dataset has successfully been processed.
+        | `processing_status` | Description |
+        | ------------------- | ----------- |
+        | `"FAILURE"` | the Dataset ran into an error during processing. See processing_status logs for more info. |
+        | `"PENDING"` | the Dataset has is still being processed. |
+        | `"SUCCESS"` | the Dataset has successfully been processed. |
+        | `"VALIDATION_FAILURE"` | An error occured during validation. See `processing_status_detail`|
```

Comment:
@brianraymor @jahilton

Reply:
What would qualify as `FAILURE` that doesn't fit into `VALIDATION_FAILURE ` or `PIPELINE_FAILURE `?